students/morgan/session03/mailroom.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+# name, donation total, number of donations and average of donations
+
+
+def main_loop():
+
+    while True:
+        answer = str(input("Select from one of these options:\n"
+              "(1) Send a Thank You\n"
+              "(2) Create a Report\n"
+              "(3) quit\n"
+              "> "))
+        if answer =='3':
+            break
+        elif answer =='1':
+            thank_you()
+        elif answer =='2':
+            print_report(donor_list)
+        else:
+            print("\nPlease type 1, 2, or 3")
+            
```

Comment:
Note that this function is only going to work if this file is run as a script. If you try to import these to another file or the command line, this function will have no idea what donor_list is, since you only define it in the `if __name__ == "__main__"` block.